src/main/java/com/auth0/json/mgmt/users/refreshtokens/RefreshTokensPage.java

```diff
@@ -0,0 +1,44 @@
+package com.auth0.json.mgmt.users.refreshtokens;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+/**
+ * This does not extend com.auth0.json.mgmt.Page<RefreshToken> because the URL only supports "next" and "take" pagination.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class RefreshTokensPage {
```

Comment:
We can use inheritance and extend generic Page<> class. For reference, refer ClientsPage class